x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java

```diff
@@ -381,18 +368,14 @@ private void testUnsuccessfulSnapshotRetention(boolean partialSuccess) throws Ex
                 assertEquals(SnapshotState.SUCCESS, snapshotInfo.state());
             });
         }
+        awaitNoMoreRunningOperations(internalCluster().getMasterName());
```

Comment:
Needed to add a new wait here because the `AbstractSnapshotIntegTestCase` does some repo consistency checks in after the tests (well worth it to have these here anyway) which will break if there's still work done in the cluster (which may be the case in this test).